core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java

```diff
@@ -24,17 +20,6 @@
 public final class MasterUfsManager extends AbstractUfsManager {
 
   /**
-   * Establishes the connection to the given UFS from master.
-   *
-   * @param ufs UFS instance
-   * @throws IOException if failed to create the UFS instance
-   */
-  public void connect(UnderFileSystem ufs) throws IOException {
```

Comment:
as discussed,  connection still made right after ufs created and when replay==false